.github/workflows/tofu-apply.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         with:
             label: dsekt-infra
             variables: |
+              ssh_user = "${{ vars.TF_SSH_USER }}"
               hcloud_token = "${{ secrets.TF_HCLOUD_TOKEN }}"
               cloudflare_api_token = "${{ secrets.TF_CLOUDFLARE_TOKEN }}"
-              ssh_user = "${{ vars.TF_SSH_USER }}"
+              vault_db_password = "${{ secrets.TF_VAULT_DB_PASSWORD }}"
```

Comment:
I'm not the biggest fan of having this password here, because now you need to have this locally as well if you need to run `tofu apply` (or should we perhaps _never_ do that locally?) and if you don't have the same one vaultwarden's database connection will just stop working until you update it in the database as well.